aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md

```diff
@@ -1,8 +1,8 @@
 ---
 uid: aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery
 title: "Continuous Integration and Continuous Delivery (Building Real-World Cloud Apps with Azure) | Microsoft Docs"
-author: MikeWasson
-description: "The Building Real World Cloud Apps with Azure e-book is based on a presentation developed by Scott Guthrie. It explains 13 patterns and practices that can he..."
+author: Rick-Anderson
+description: "Building Real World Cloud Apps with Azure. An e-book is based on a presentation developed by Scott Guthrie, which explains 13 patterns and practices that can he..."
```

Comment:
Remove comma after "Guthrie". (We should also research whether to go for "that" or "which".)